polkadot-parachains/src/command.rs

```diff
@@ -42,7 +41,11 @@ use sp_core::hexdisplay::HexDisplay;
 use sp_runtime::traits::Block as BlockT;
 use std::{io::Write, net::SocketAddr};
 
-const DEFAULT_PARA_ID: u32 = 2015;
+const LOCAL_PARA_ID: u32 = 2000;
```

Comment:
this was overdue. as we can't select our paraid ourselves, here are all the reserved id's